src/app/components/audio-recordings/pages/download/download.component.html

```diff
@@ -309,13 +309,19 @@ <h3>3. Unblock the downloader script</h3>
 
   <h3>4. Open PowerShell</h3>
 
-  <p>
-    <a
-      href="https://www.groovypost.com/howto/open-command-window-terminal-window-specific-folder-windows-mac-linux/"
-      >Open a PowerShell shell</a
-    >
-    in the folder containing the download script
-  </p>
+  <ul>
+    <li>
+      <a
+        href="https://www.groovypost.com/howto/open-command-window-terminal-window-specific-folder-windows-mac-linux/"
+      >
+        Open a Powershell shell
+      </a>
+      in the folder containing the download script
+    </li>
+    <li>
+      If you are on Linux/MacOS, run the <code>pwsh</code> command from your terminal/command line to start PowerShell and press <kbd>enter</kbd>.
```

Comment:
```suggestion
      If you are on Linux/MacOS, run the <code>pwsh</code> command from your terminal to start PowerShell.
```

Reply:
Looks good, I'll be squashing and merging shortly